.azure/Makefile

```
@@ -1,15 +1,14 @@
+PLUME_SCRIPTS ?= .plume-scripts
+ifeq (,$(wildcard ${PLUME_SCRIPTS}))
+dummy := $(shell git clone --depth=1 -q https://github.com/plume-lib/plume-scripts.git ${PLUME_SCRIPTS})
+endif
+
 all default: azure-pipelines.yml
```

checkmake (0.3.2)

[warning] Target "all default" should be declared PHONY.

(phonydeclared)